src/main/res/values/strings.xml

```diff
@@ -375,8 +375,8 @@
         <item quantity="other">Delete %d messages here and on the server?</item>
     </plurals>
     <plurals name="ask_delete_messages_simple">
-        <item quantity="one">Delete %d message?</item>
-        <item quantity="other">Delete %d messages?</item>
+        <item quantity="one">Delete %d message on all devices?</item>
```

Comment:
I think it is better if it says "on all YOUR devices" since otherwise it looks like this delete in all devices of all people